src/tools/reporters/PublicReporter.ts

```diff
@@ -53,5 +58,12 @@ export class PublicReporter {
         proseWrap: "always",
       }),
     );
+
+    Reporter!.notifyStorageAboutContracts(contracts);
+  }
+
+  public static disableShortenAddress(): typeof PublicReporter {
```

Comment:
Why do we ever need shortened addresses?

Reply:
Deleted this functionality